framework/functions.php

```diff
@@ -257,12 +257,13 @@ function fw_menu_output ( $menu, $level, $type, $classes ) {
 	echo '">';
 
 	foreach ( $menu as $item ) {
-	
+
 		echo '<li class="';
 
+		$base_current_url = strpos($GLOBALS['vars']['current_url'], '?') ? substr($GLOBALS['vars']['current_url'], 0, strpos($GLOBALS['vars']['current_url'], '?')) : $GLOBALS['vars']['current_url'];
```

Comment:
Please break this condition down on multiple so it's more readable. It's complicated to read with the imbricated functions. You can use a $pos var if necessary to make it more easy to understand.
Add spaces before and after parenthesis to respect the current syntax of the file.
Also, you should work with $GLOBALS['vars']['current_url'] after verifying if it is set. Right now, you are assuming it is, and checking if it is set on the next line.

Reply:
@domcode-ew I updates this piece of code. Can you review it again? Thanks!